src/components/ProgressBar/index.tsx

```diff
@@ -119,34 +130,38 @@ const LinearProgressComponent: React.FunctionComponent<ProgressBarComponentProps
     );
 };
 
-const CircularProgressWithLabel: React.FunctionComponent<ProgressBarComponentProps> = ({
-    value,
-    displayValue,
-    ...props
-}) => {
+const CircularProgressWithLabel: React.FunctionComponent<
+    ProgressBarComponentProps & Omit<CircularProgressProps, 'variant'>
+> = ({ value, displayValue, ...props }) => {
     const classes = useStyles();
 
     return (
         <Box position="relative" display="inline-flex">
             <CircularProgress
-                variant="static"
+                value={100}
+                variant={'static'}
+                classes={{ colorPrimary: classes.circularColorBottom }}
+                {...props}
+            />
+            <CircularProgress
+                variant={'static'}
```

Comment:
Do you need 2 CircularProgress here?

Reply:
Understand now :)

Reply:
Yeh, One is overlayed on top of the other.

Basically the grey circle is completely colored and the blue circle (ontop) is where you are in relation to 100%. This way is looks better visually, otherwise for 0% case you would see nothing.